### DISCUSSION TITLE
How to get a list of all users logged in/online ?

### DISCUSSION THREAD

**Discussion by gevera** [pocketbase/pocketbase#1706] [Q&A] [answered]
What would be the best approach of displaying / getting the list of all the users that are online. Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
There is no "presence" like built-in functionality. You can also check https://github.com/pocketbase/pocketbase/discussions/845 and https://github.com/pocketbase/pocketbase/discussions/1607.

You can get a list with all realtime connected clients by accessing `app.SubscriptionsBroker().Clients()`
Note that a single user may have more than one client connection, eg. opened in a separate tab/window. You can get the current client auth record (if authorized) similarly to the [request context](https://pocketbase.io/docs/router/#get-logged-admin-or-app-user), eg. `app.SubscriptionsBroker().Clients()[0].Get(apis.ContextAuthRecordKey)`.

**Reply by gevera** [reply]
Thank you for prompt reply.
